### ISSUE TITLE
org-element-at-point cannot be used in non-Org buffer

### ISSUE THREAD

**Issue by cvanelteren** [nim-lang/nim-mode#246]
After recent upgrade of my doom emacs I am experiencing this bug when activating nim-mode. Similar to https://github.com/alphapapa/org-web-tools/issues/56. Is there a plan in the works for a fix?

<details> <summary> info  </summary>

```
nim-mode                       20211102.917   installed             A major mode for the Nim programming language
org                            9.6.9          available    gnu      Outline-based notes management and organizer T
```
 </details>

**Comment by Vindaar** [comment]
There's a fix already here: https://github.com/nim-lang/nim-mode/pull/245

I'll see about getting that merged. Feel free to check out that PR locally to see if it fixes the issue for you (I haven't done so yet).

**Comment by Vindaar** [comment]
This should hopefully be fixed with #245 merged. Feel free to reopen if the issue persists.
